app/app.go

```diff
@@ -677,3 +678,25 @@ func (a *App) SetServer(srv *Server) {
 func (a *App) GetT() goi18n.TranslateFunc {
 	return a.t
 }
+
+func (a *App) DBHealthCheckWrite() error {
```

Comment:
Why this new functions are needed? I don't see any changes in the code calling this new functions.